share/ShareViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  ShareViewController.swift
+//  share
+//
+//  Created by aanorbel on 4/21/22.
+//  Copyright © 2022 OONI. All rights reserved.
+//
+
+import UIKit
+import Social
+import CoreServices
+/*
+
+class ShareViewController: SLComposeServiceViewController {
+
+    override func isContentValid() -> Bool {
+        // Do validation of contentText and/or NSExtensionContext attachments here
+        return true
+    }
+
+    override func didSelectPost() {
+        // This is called after the user selects Post. Do the upload of contentText and/or NSExtensionContext attachments.
+    
+        // Inform the host that we're done, so it un-blocks its UI. Note: Alternatively you could call super's -didSelectPost, which will similarly complete the extension context.
+        self.extensionContext!.completeRequest(returningItems: [], completionHandler: nil)
+    }
+
+    override func configurationItems() -> [Any]! {
+        // To add configuration options via table cells at the bottom of the sheet, return an array of SLComposeSheetConfigurationItem here.
+        return []
+    }
+
+}
+*/
+
+
+class ShareViewController: UIViewController {
```

Comment:
Please, add documentation for this class explaining how it interacts with the rest of the app.